cg/meta/workflow/rnafusion.py

```diff
@@ -140,8 +140,12 @@ def write_samplesheet(self, case_id: str, strandedness: str, dry_run: bool = Fal
 
         for link in case_obj.links:
             sample_metadata: List[str] = self.gather_file_metadata_for_sample(link.sample)
-            fastq_r1: List[str] = NextflowAnalysisAPI.extract_read_files(1, sample_metadata)
-            fastq_r2: List[str] = NextflowAnalysisAPI.extract_read_files(2, sample_metadata)
+            fastq_r1: List[str] = NextflowAnalysisAPI.extract_read_files(
+                read_nb=1, metadata=sample_metadata
```

Comment:
What does nb in ´read_nb´ stand for?

Reply:
`read_number` for fastq files: 1 for forward and 2 for reverse

Reply:
Would be great to be explicit and spell it out if possible or even have separate args for forward and reverse to avoid having to know what 1 and 2 stands for in this context

Reply:
done!